kie-plugins-testing/src/test/projects/kjar-8-multimodule/modA/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <name>Drools-Test-A</name>
+  <description>Testing drools executable model capability</description>
+
+  <artifactId>kie-maven-plugin-test-kjar-8-modA</artifactId>
+  <version>1.0.0.Final</version>
```

Comment:
Child modules don't need to have version defined if they have a parent. So you could remove this line, also in other child modules.